src/Hazelcast.Net.Tests/Networking/AddressProviderTests.cs

```diff
@@ -235,10 +273,10 @@ public void CreateMapReInvokedIfMappingFails()
 
             Assert.That(count, Is.EqualTo(0));
 
-            var addresses1 = addressProvider.GetAddresses();
+            var unused1 = addressProvider.GetAddresses();
             Assert.That(count, Is.EqualTo(1));
 
-            var address1 = addressProvider.Map(NetworkAddress.Parse("192.168.0.2"));
+            var unused2 = addressProvider.Map(NetworkAddress.Parse("192.168.0.2"));
             Assert.That(count, Is.EqualTo(2)); // failed Map = tried again
```

Comment:
Just as a note, moq library has nice abilities to count and validate such invocations.